hca/dss/__init__.py

```diff
@@ -142,39 +174,39 @@ def _download_file(self,
                 if e.errno != errno.EEXIST:
                     raise
         with atomic_write(dest_path, mode="wb", overwrite=True) as fh:
-            if size == 0:
-                logger.info("%s", "File {}: CREATED (empty). Stored at {}.".format(file_uuid, dest_path))
+            if dssfile.size == 0:
```

Comment:
Nit: `dss_file` since the camel case class name is `DSSFile` signifying two separate words, as in "DSS file".

Reply:
Done